pkgs/applications/science/electronics/openhantek6022/udev-rules.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/utils/udev_rules/60-hantek.rules b/utils/udev_rules/60-hantek.rules
+index 77b34b8..e91ea0e 100644
+--- a/utils/udev_rules/60-hantek.rules
++++ b/utils/udev_rules/60-hantek.rules
+@@ -1,16 +1,16 @@
+ # rules for OpenHantek6022 (DSO program) as well as Hankek6022API (python tools)
+ 
+ # Hantek DSO-6022BE, without FW, with FW
+-SUBSYSTEM=="usb", ACTION=="add", ENV{DEVTYPE}=="usb_device", ATTRS{idVendor}=="04b4", ATTRS{idProduct}=="6022", TAG+="uaccess", TAG+="udev-acl"
```

Comment:
Any reason not to rely on `uaccess`? `systemd-logind` should automatically give the user the right permissions when they are logged in. Did you check why it doesn't work?

Also, these simple replacements are better done with `substituteInPlace`. Then the patch does not have to be updated when new devices are added.

Reply:
I thought, reloading rules and unplugging the USB device would trigger the new rules already. I did not test starting a new session. I will check

Reply:
Yep, checked, after a fresh restart, it works with the default udev rules